### PR TITLE
Further Modernize Demos

### PR DESCRIPTION
**Pull request by stephencelis**
Just some mentally untaxing work to round out the day.

### CODE REVIEW

File: Examples/CaseStudies/CaseStudies.xcodeproj/xcshareddata/xcschemes/tvOSCaseStudies.xcscheme [hunk 1]
```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (stephencelis) [inline Examples/CaseStudies/CaseStudies.xcodeproj/xcshareddata/xcschemes/tvOSCaseStudies.xcscheme:1]: We were missing this shared scheme.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -34,9 +34,7 @@ jobs:
     strategy:
       matrix:
         xcode:
-          - '12.4'
-          - '12.5.1'
-          - '13.1'
+          - '13.2'
```
[the remark this answers is not in the document]
Comment (iampatbrown) [reply .github/workflows/ci.yml:37]: This might be relevant dc77e7a from #859

[the remark this answers is not in the document]
Comment (iampatbrown) [reply .github/workflows/ci.yml:37]: Actually I think it was just something that would have needed an `@available` check.

**Review by mbrandonw — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (stephencelis) [reply .github/workflows/ci.yml:37]: @iampatbrown Ha so nothing to add? Gonna merge now but could bring more in in another PR later.

[the remark this answers is not in the document]
Comment (iampatbrown) [reply .github/workflows/ci.yml:37]: @stephencelis no worries :)

**Pull request merged**